package.json

```diff
@@ -9,19 +9,20 @@
     "cors": "^2.8.5",
     "dotenv": "^16.4.5",
     "express": "^4.18.3",
+    "jsonwebtoken": "^9.0.2",
     "knex": "^3.1.0",
     "mysql2": "^3.9.2",
     "pg": "^8.11.3",
     "postgres": "^3.4.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.22.3",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
     "start-server": "node src/backend/server.js",
-    "start-frontend": "react-scripts start --port 3003",
-    "start": "npm run start-server & npm run start-frontend",
```

Comment:
Was there a specific reason for removing this? maybe we should update the readme for startup approach

Reply:
We tried to unify the frontend and the backend start and make it run in different ports, but talking with other of the mentors we decided to keep it separate 

  "start-server": "node src/backend/server.js",
    "start-frontend": "react-scripts start "